appcues/build.gradle

```diff
@@ -11,7 +11,7 @@ apply from: 'copy-dependencies.gradle'
 apply from: 'dokka.gradle'
 
 ext.room_version = '2.5.2'
-ext.compose_version = '1.5.2'
+ext.compose_version = '1.6.0'
```

Comment:
any value in us moving to the BOM approach for the compose packages at this point?

Reply:
I'm old school, I rather know which version we are on, also probably makes it easier for customers who are consulting our code maybe ? not sure